src/exporter/DaemonMetricCollector.cc

```diff
@@ -285,18 +288,37 @@ DaemonMetricCollector::get_labels_and_metric_name(std::string daemon_name,
   std::string new_metric_name;
   labels_t labels;
   new_metric_name = metric_name;
+  std::string ceph_daemon_prefix = "ceph-";
```

Comment:
We have discover this with "ceph_osd_numpg" metrics. Are you sure this is applicable to all the rest of metrics?

Reply:
> We have discover this with "ceph_osd_numpg" metrics. Are you sure this is applicable to all the rest of metrics?

Yes. I've attached the screenshot as well. For fixing rgw related panels we need additional and major changes which I'm addressing here https://github.com/ceph/ceph/pull/52325/files. 
And for addressing the testing we already have a  tracker and as discussed before we agreed to address it in a separate PR.